imagedephi/gui.py

```diff
@@ -28,4 +31,8 @@ def select_directory(request: Request, path: str = "/"):
 
 @app.post("/directory_selection/")
 def selection(directory: str = Form()):  # noqa: B008
```

Comment:
```suggestion
def selection(directory: pathlib.Path = Form()):  # noqa: B008
```

We should be able to make FastAPI do the type coercion for us as it parses the form. You could then remove the explicit coercion on the next line.

Reply:
fixed in cbdc6e68684f30990e1b2f5e8764d59ac3fbbebf